src/main/java/wooteco/subway/line/dao/LineDao.java

```diff
@@ -20,25 +19,25 @@
     private JdbcTemplate jdbcTemplate;
     private SimpleJdbcInsert insertAction;
 
-    public LineDao(JdbcTemplate jdbcTemplate, DataSource dataSource) {
+    public LineDao(JdbcTemplate jdbcTemplate) {
         this.jdbcTemplate = jdbcTemplate;
-        this.insertAction = new SimpleJdbcInsert(dataSource)
-                .withTableName("line")
+        this.insertAction = new SimpleJdbcInsert(jdbcTemplate)
+                .withTableName("LINE")
                 .usingGeneratedKeyColumns("id");
     }
 
     public Line insert(Line line) {
         Map<String, Object> params = new HashMap<>();
-        params.put("id", line.getId());
         params.put("name", line.getName());
         params.put("color", line.getColor());
+        params.put("extra_fare", line.getExtraFare());
 
         Long lineId = insertAction.executeAndReturnKey(params).longValue();
-        return new Line(lineId, line.getName(), line.getColor());
+        return new Line(lineId, line.getName(), line.getColor(), line.getExtraFare());
```

Comment:
```suggestion
        return findById(lineId);
```
는 어떤가요 이유는 DM 으로 나눴던 이야기랑 비슷해요.

Reply:
이렇게 변경하니 extractSections에서 result가 empty일 때 Collections.EMPTY_LIST를 반환하도록 했었는데, 이가 Immutable List여서 Abstract List에 add하지 못하는 문제를 확인할 수 있었네요! 
아무쪼록 변경하고 immutable List 대신 mutableList를 반환하도록 수정까지 완료하였습니다~~